src/main/java/com/sda/games/wheelOfFortune/methods/Randoms.java

```diff
@@ -0,0 +1,23 @@
+package com.sda.games.wheelOfFortune.methods;
+
+import com.sda.games.wheelOfFortune.dao.CategoryDao;
+import com.sda.games.wheelOfFortune.dao.WordsDao;
+
+import java.util.Random;
+
+public class Randoms {
+    static int generateRNCategory() {
+        CategoryDao categoryDao = new CategoryDao(WheelGame.hibernateFactory);
+        int max = categoryDao.getAllCount();
+        Random random = new Random();
+        return random.nextInt(max) + WheelGame.MIN;
+    }
+
+    static int generateRNWord(int drawnNumberCategory) {
+        WordsDao wordsDao = new WordsDao(WheelGame.hibernateFactory);
+        int max = wordsDao.getAllCountWordsFromCategory(drawnNumberCategory);
```

Comment:
sie przypomne ze tu by bylo lepiej wrocic liste ale moze byc osobny PR na to :)

Reply:
To, to już materiał na change requesta :D